### PR TITLE
Add router + Landing Page

### PR DESCRIPTION
**Pull request by jbenning97**
* Added Router
* The landing route is called Home
* The calendar route is called Main
* Strictly UI, no functionality has been implemented

### CODE REVIEW

File: src/routes/Home.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      <div className="left-container">
+        <img src={logo} className="logo" alt="" />
+      </div>
+      <div className="right-container">
+        <StyledCard>
+          <h1>Schedule Courses</h1>
+          <h3>Time Table Builder for UVic Students</h3>
+          <form id="signin">
+            <TextField id="email" label="Email" />
+            <TextFieldPwd id="password" label="Password" type="password" />
```
Comment (kerfootj) [inline src/routes/Home.tsx]: Can you add autoComplete="username" to the email TextField and autoComplete="current-password" to the password TextField 

Reference: https://material-ui.com/api/text-field/

File: src/routes/Home.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import React from 'react';
```
Comment (kerfootj) [inline src/routes/Home.tsx]: Can you rename the routes folder to pages. Trying to some what follow [atomic design](https://bradfrost.com/blog/post/atomic-web-design/). At least as far as organizing components.

File: src/routes/Home.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  marginTop: '2px',
+  textTransform: 'none',
+  width: 'fit-content',
+});
+
+const TextFieldPwd = styled(TextField)({
+  marginTop: '8px',
+});
+
+const SigninBtn = styled(Button)({
```
Comment (kerfootj) [inline src/routes/Home.tsx]: Can you change the hover color on the button. The text is hard to read when hovering over.

**Review by kerfootj — COMMENTED** [review]
The page looks great!

File: src/routes/Home.tsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+});
+
+const Home: React.FC = () => {
+  return (
+    <div className="main-container">
+      <div className="left-container">
+        <img src={logo} className="logo" alt="" />
+      </div>
+      <div className="right-container">
+        <StyledCard>
```
Comment (kerfootj) [inline src/routes/Home.tsx]: We'll probably want to extract this to it's own component so we can reuse the same page but swap out Sign In or Sign Up forms

File: public/index.css [hunk 5]
```diff
@@ -0,0 +1,8 @@
+html, body, #root {
+  height: 100%;
+  font-family: 'Open Sans', sans-serif;
+}
+
+body {
+  margin: 0;
+}
```
Comment (BraidenCutforth) [inline public/index.css]: Nitpick but I think it's standard practice to have a newline at the end of the file.

File: public/index.html [hunk 6]
```diff
@@ -24,6 +24,8 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="stylesheet" href="index.css">
```
Comment (BraidenCutforth) [inline public/index.html:27]: Whatever .tsx file has the component that relies on the css file should import it in that fashion.
I believe the syntax is:
```typescript
import 'path/to/index.css'
```
noting to `../` up the tree from the file as necessary until @kerfootj changes the import style

File: src/App.tsx [hunk 7]
```diff
@@ -1,11 +1,20 @@
...
 
 const App: React.FC = () => {
   return (
-    <div>
-      <Calendar />
-    </div>
+    <Router>
+      <Switch>
+        <Route path="/main">
+          <Main />
```
Comment (BraidenCutforth) [inline src/App.tsx:11]: What is Main vs Home?

File: src/routes/Home.css [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+  flex-direction: column;
+}
+
+.signup-container {
+  margin-top: 14px;
+}
+
+.signup-text {
+  font-size: 0.875rem;
+}
```
Comment (BraidenCutforth) [inline src/routes/Home.css]: newline

File: src/routes/Home.tsx [hunk 9]
```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { styled } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card';
+import Divider from '@material-ui/core/Divider';
+import TextField from '@material-ui/core/TextField';
+import logo from '../assets/logo.png';
+import './Home.css';
```
Comment (BraidenCutforth) [inline src/routes/Home.tsx]: Looks like you already used this import style :+1:

File: src/routes/Home.tsx [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+const ForgotPwdBtn = styled(Button)({
```
Comment (BraidenCutforth) [inline src/routes/Home.tsx]: This is a way different css style then Passr uses but it looks WAY more elegant that what we are doing. Let's chat and maybe you can point me in the right direction.

File: src/routes/Home.tsx [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+  maxWidth: '400px',
+  padding: '0 38px 24px 38px',
+  textAlign: 'center',
+});
+
+const StyledDivider = styled(Divider)({
+  width: '80px',
+});
+
+const Home: React.FC = () => {
```
Comment (BraidenCutforth) [inline src/routes/Home.tsx]: Beautiful use of composition 🔥

File: src/routes/Main.tsx [hunk 12]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Calendar from '../atoms/Calendar/Calendar';
+
+const Main: React.FC = () => {
```
Comment (BraidenCutforth) [inline src/routes/Main.tsx]: I see what Main is now. I am wondering if there is a better name, but I also can't think of one..

**Review by BraidenCutforth — CHANGES_REQUESTED** [review]

File: src/routes/Main.tsx (hunk 12 above)
Comment (kerfootj) [reply src/routes/Main.tsx]: I think it will change. I'm just not sure what to right now

File: src/App.tsx (hunk 7 above)
Comment (jbenning97) [reply src/App.tsx:11]: As of now, Main is the page with the calendar, and Home is the landing/sign-in page. Open to renaming them.

**Review by kerfootj — APPROVED** [review]
Looks good

**Pull request merged**